src/components/Carousel/Carousel.module.css

```diff
@@ -0,0 +1,28 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
```

Comment:
You might want to add 
`justify-content: center;`
if looking for centring the component on the y-axis (already centred on the x-axis).

Reply:
Any external stylings of the component (e.g. the y-axis positioning) should probably not be applied in the component itself, because we (as users of the component) should be able to position it wherever needed from the outside rather than changing the internal styling.

For example, when we create a `Login` component, we can pass `className` to it and position it on screen wherever we want on our screen. This `className` will also allow us to provide any `max-width` property so that we can adapt the component for mobile/desktop screens from the outside.

Reply:
Okay that makes sense!